File: src/main/java/io/github/faecraft/heartshapedbox/body/BodyPartSide.java
package io.github.faecraft.heartshapedbox.body;

import java.util.Locale;

interface lowerCaseEnumName {
    String lowerName();
}

public enum BodyPartSide implements lowerCaseEnumName {
    CENTER,
    LEFT,
    RIGHT;
    
    @Override
    public String lowerName() {
        return name().toLowerCase(Locale.ROOT);
    }
}

File: src/main/kotlin/io/github/faecraft/heartshapedbox/body/BodyPartSide.kt
package io.github.faecraft.heartshapedbox.body

import java.util.*

internal interface LowerCaseEnumName {
    fun lowerName(): String
}

public enum class BodyPartSide : LowerCaseEnumName {
    CENTER, LEFT, RIGHT;

    override fun lowerName(): String = name.toLowerCase(Locale.ROOT)
}
